techmatters/webchat
# Bypass Recaptcha for automated testing

## Description
- Added `data-bypassCaptcha` as a html script attribute to help bypass recaptcha for uptime monitoring
- Sends e2eTestMode attribute with message

This has been tested in aselo development.

### Checklist



### Related Issues
Fixes # [CHI-2047](https://tech-matters.atlassian.net/browse/CHI-2047)

### Verification steps
- Check locally that reCaptcha is bypassed
- This branch has been deployed to E2E webchat -  https://tl-public-chat-e2e-dev.s3.amazonaws.com/e2e-chat-development.html. Check that the script for html has the `bypasscaptcha` flag as a data attribute

[CHI-2047]: https://tech-matters.atlassian.net/browse/CHI-2047?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
